README.md

```diff
@@ -21,6 +21,9 @@ This repository is made up of two separate Unity projects. If you would like to
 * ArcGISMaps SDK for Unity
 * A VR Headset and the necessary software to run through Desktop Mode
 
+### Setting Up the XR Sample Project
```

Comment:
I think it would be more appropriate to put this between `Instructions` and `Requirements`.